quic/s2n-quic-transport/src/space/rx_packet_numbers/mod.rs

```diff
@@ -5,6 +5,8 @@ mod ack_eliciting_transmission;
 mod ack_manager;
 pub(crate) mod ack_ranges;
 mod ack_transmission_state;
+#[allow(dead_code)]
+mod pending_ack_ranges;
```

Comment:
these technically aren't rx packet numbers. They're pending tx. we may want to restructure things to just be `transport/src/packet_numbers`?

Reply:
was actually thinking we could move/rename to `/transport/src/ack` which now exists

Reply:
its a large diff. lemme know if you want me to make it a separate PR https://github.com/aws/s2n-quic/pull/1281/commits/86444593fc21f41592e094b32e23349d3f273995